include/sorting_network.h

```diff
@@ -91,7 +91,6 @@ class SortingNetwork
    *  @return the combined vector
    */
   TermVec merge(const TermVec & sorted1, const TermVec & sorted2) const;
-
```

Comment:
It would be better not to include this change in this PR since it is in an unrelated file.